package.json

```
@@ -10,6 +10,7 @@
     "bootstrap": "3.4.1",
     "chart.js": "2.9.4",
     "classnames": "2.2.6",
+    "country-list": "^2.2.0",
```

nit: use exact versioning to fix the import to what we know works:
```suggestion
    "country-list": "2.2.0",
```